java_console/io/src/main/java/com/rusefi/io/DfuHelper.java

```diff
@@ -34,15 +34,8 @@ public static boolean sendDfuRebootCommand(JComponent parent, String signature,
         RusEfiSignature controllerSignature = SignatureHelper.parse(signature);
         String fileSystemBundleTarget = BundleUtil.getBundleTarget();
         if (fileSystemBundleTarget != null && controllerSignature != null) {
-/*
-todo: fix https://github.com/rusefi/rusefi/issues/5016
             String signatureWithPrefix;
-            if ("all".equals(s.getBundleTarget())) {
-                signatureWithPrefix = PREFIX;
-            } else {
-                signatureWithPrefix = PREFIX + "_" + s.getBundleTarget();
-            }
-*/
+            signatureWithPrefix = PREFIX + "_" + controllerSignature.getBundleTarget();
```

Comment:
@chuckwagoncomputing are you adding an unused variable here?

Reply:
I guess so. I just preserved it without checking to see if it was ever used.